### PR TITLE
inm/#1813 output format changed to gzip for 2 output files

### LINKED ISSUE #1813: gzip these two csvs
image

### PR DESCRIPTION
**Pull request by irishwithaxe**
closes #1813 

for EuclideanVsLengthAttribute and tazRideHailSurgePriceLevel

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/2169)

### CODE REVIEW

**Comment by irishwithaxe** [comment]
test!

File: src/main/java/beam/analysis/plots/GraphSurgePricing.java [hunk 1]
```diff
@@ -445,8 +446,8 @@ private void writePriceSurgeCsv(double[][] dataset, List<String> categoriesList,
     }
 
     private void writeTazCsv(Map<String, double[][]> dataset) {
-
-        try (BufferedWriter out = new BufferedWriter(new FileWriter(new File(surgePricingAndRevenueWithTaz)))) {
+        // try (BufferedWriter out = new BufferedWriter(new FileWriter(new File(surgePricingAndRevenueWithTaz)))) {
```
Comment (JustinPihony) [inline src/main/java/beam/analysis/plots/GraphSurgePricing.java]: No need to keep commented code around - let history handle any questions.

Comment (irishwithaxe) [reply src/main/java/beam/analysis/plots/GraphSurgePricing.java]: agree

**Comment by irishwithaxe** [comment]
Test!

**Comment by JustinPihony** [comment]
@irishwithaxe Any reason this can't be merged?

**Pull request merged**